src/main/java/com.marketdata.api/MarketDataController.java

```diff
@@ -1,7 +1,5 @@
 package com.marketdata.api;
 
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
```

Comment:
Unused imports. The shortcut for formatting code in IntelliJ is ctl-shift-alt-L.
https://www.baeldung.com/java-clean-code

Reply:
The shortcut for organizing imports is Control Shift O.  It's a good idea to do this periodically.